rest-service/manager_rest/rest/resources_v3_1/deployments.py

```diff
@@ -575,21 +575,23 @@ def put(self, group_id):
             'deployments_from_group': {'optional': True},
         })
         sm = get_storage_manager()
-        try:
-            group = sm.get(models.DeploymentGroup, group_id)
-        except manager_exceptions.NotFoundError:
-            group = models.DeploymentGroup(
-                id=group_id,
-                description=request_dict.get('description'),
-                created_at=datetime.now()
-            )
         with sm.transaction():
+            try:
+                group = sm.get(models.DeploymentGroup, group_id)
```

Comment:
Do we deliberately not update the description when performing a put on a deployment group?

Reply:
we do, in `_set_group_attributes`. You're right, no point doing it twice.